resources/views/site/airport/index.blade.php

```diff
@@ -10,14 +10,14 @@
                         <div class="col-md-6 text-center">
                             <ul class="list-unstyled">
                                 @foreach($airports->first() as $airport)
-                                    <li>@if($airport->major)<strong>@endif<a href="{{route('airport.view', $airport->icao)}}">{{$airport->name}} ({{$airport->icao}} / {{$airport->iata}})</a>@if($airport->major)</strong>@endif</li>
+                                    <li>@if($airport->major)<strong>@endif<a href="{{route('site.airport.view', $airport->icao)}}">{{$airport->name}} ({{$airport->icao}} / {{$airport->iata}})</a>@if($airport->major)</strong>@endif</li>
```

Comment:
There isn't anything wrong with how we are doing this but I feel we could utilise a span and apply a css class (font-weight-bold) conditionally, rather than fanny around with the <strong> tag.

E.g. `<span class="@if ($airport->major) font-weight-bold @endif">`

Reply:
`font-weight-bold` is not available in bootstrap 3 and the accepted method is using `<strong>` (ref: https://getbootstrap.com/docs/3.3/css/)

Reply:
Serves me right for reading the v4 docs. Whoops!